website/js/index.js

```diff
@@ -1114,9 +1114,12 @@ var VariantDetail = React.createClass({
                 </Row>
 
                 <Row>
-                    <Col md={12} className="variant-literature-col">
-                        <LiteratureTable maxRows={10} variant={variant} hideEmptyItems={this.state.hideEmptyItems} />
-                    </Col>
+                    { this.props.mode === "research_mode" && (
```

Comment:
I'd prefer to hide the `<Row>` element as well, just in case it takes up space even if there aren't any columns in it.

Reply:
good call, just updated